server/app/models/Version.java

```diff
@@ -171,4 +179,29 @@ public boolean removeTombstoneForQuestion(Question question) {
     }
     return this.tombstonedQuestionNames.remove(question.getQuestionDefinition().getName());
   }
+
+  @VisibleForTesting
+  public boolean addTombstoneForProgramForTest(Program program) {
```

Comment:
suggestion (blocking): since this is public, it doesn't need the testing annotation -- either make it private or remove the annotation

Reply:
Done.